src/player.cpp

```diff
@@ -264,8 +266,15 @@ unsigned int Player::play(
         return 0;
 
     ActiveSound *sound = s->get();
-    SoLoud::handle newHandle = soloud.play(*sound->sound.get(), volume, pan, paused, 0);
-    sound->handle.emplace_back(newHandle);
+    SoLoud::handle newHandle = soloud.play(*sound->sound.get(), volume, pan, looping ? true : paused, 0);
```

Comment:
> `looping ? true : paused`

Does this mean that setting `looping` automatically makes the sound paused?

UPDATE: Oh, I get it, you're starting it as paused because when it's looping, you want to be able to set up the looppoint etc. below. I'd add a comment explaining it, so as not confuse future code reader.

Reply:
yes true, I write some explaination.
The problem that make me to do that, is the fact that, even if I set a `loopingStartAt`, the first time the sound plays it plays from the beginning. From the 2nd time the starting point is used